locale/fr_CA/locale.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/fr_CA/locale.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```

Comment:
All copyrights should be to Simon Fraser University, not Simon Fraser University Library; also update all copyright dates to end in 2018.

Reply:
strange...
We are doing the translation with the translator plugin, I'm curious about where these old copyright information are taken.

following are my corrections.

Reply:
Hmm, I probably need to update the translator plugin -- I'll do this when OJS 3.1.1 is released (fairly soon). Thanks for this!